src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/nonassociations/NonAssociationsReconciliationService.kt

```diff
@@ -271,7 +283,7 @@ class NonAssociationsReconciliationService(
     dpsReasonEnum: NonAssociation.Reason,
   ): Boolean {
     val t = translateToRolesAndReason(reason, recipReason)
-    return t.first != role1 || t.second != role2 || ((reason == "BUL" || reason == "RIV") && t.third != dpsReasonEnum)
+    return t.first != role1 || t.second != role2 // too strict: || ((reason == "BUL" || reason == "RIV") && t.third != dpsReasonEnum)
```

Comment:
Leaving out the test on the dps reason as this is not important according to Mike W